Candidate: CVE-2025-32414
PublicDate: 2025-04-08 03:15:00 UTC
References:
 https://www.cve.org/CVERecord?id=CVE-2025-32414
Description:
 In libxml2 before 2.13.8 and 2.14.x before 2.14.2, out-of-bounds memory
 access can occur in the Python API (Python bindings) because of an
 incorrect return value. This occurs in xmlPythonFileRead and
 xmlPythonFileReadRaw because of a difference between bytes and characters.
Ubuntu-Description:
Notes:
 mdeslaur> as of 2025-04-09, the fix is still being discussed in the bug
Mitigation:
Bugs:
 https://gitlab.gnome.org/GNOME/libxml2/-/issues/889
Priority: medium
Discovered-by:
Assigned-to:
CVSS:

Patches_libxml2:
upstream_libxml2: needs-triage
esm-infra-legacy/trusty_libxml2: needs-triage
esm-infra/xenial_libxml2: needs-triage
esm-infra/bionic_libxml2: needs-triage
focal_libxml2: needs-triage
jammy_libxml2: needs-triage
noble_libxml2: needs-triage
oracular_libxml2: needs-triage
devel_libxml2: needs-triage
